x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/task/agent/sub_graphs/translate_rule/nodes/translation_result/translation_result.ts

```diff
@@ -12,36 +12,37 @@ import {
 } from '../../../../../../../../../../common/siem_migrations/constants';
 import type { GraphNode } from '../../types';
 
-export const translationResultNode: GraphNode = async (state) => {
-  // Set defaults
-  const elasticRule = {
-    title: state.original_rule.title,
-    description: state.original_rule.description || state.original_rule.title,
-    severity: DEFAULT_TRANSLATION_SEVERITY,
-    risk_score: DEFAULT_TRANSLATION_RISK_SCORE,
-    ...state.elastic_rule,
-  };
+export const getTranslationResultNode = (): GraphNode => {
+  return async (state) => {
```

Comment:
I don't understand the reason of this change

Reply:
Initially I was passing in the telemetry client there, but decided it was better in the task client for results because of the duration.
Once I was going to revert it I kept it like that to have it be consistent with all the other nodes. I can revert it to its original state if you want 👍